Interface.md

```diff
@@ -42,5 +42,13 @@ services through the exploitation of standard protocols and the harmonizing of s
 | **AUTH\#1** |  Authentication interfaces should support the [OpenID Connect](https://openid.net/developers/how-connect-works/) Protocol. |
 | **AUTH\#2** |  HTTP requests (e.g. for data access) subject to authorization shall include the user token with `claims` in [JWT](https://datatracker.ietf.org/doc/html/rfc7519) format returned by the Authentication interface with every API request as a bearer token in the HTTP authorization header. |
```

Comment:
I haven't used JWT, but again, I think it's more for user-facing applications.

So, this method is useful when using a website, but less useful when programmatically accessing thousands or files via HTTP.

Reply:
JWT is just a way to format tokens. That can in principle also be used programmatically, the more difficult part if rather to establish long-term m2m authentication via OIDC. Recommendation around that would be good. The recommendations right now seem to require human interaction and are more targeted towards observed systems.